core/templates/dev/head/library/library.html

```diff
@@ -59,10 +59,15 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
           <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
             <div class="row oppia-library-group-header-container">
               <h2 class="oppia-library-group-header">
-                <a ng-click="showAllResultsForCategories(group.categories)" translate="<[group.translationId]>">
+                <a ng-if="libraryGroups[$index].header != 'Featured Activities'"
```

Comment:
I will try to use i18n id instead of the string. I agree that this is a bit confusing. Could you just give me a start on how to go about this problem

Thanks